src/data_info/services/uuids.clj

```diff
@@ -56,29 +40,6 @@
         (mapv (partial stat/decorate-stat cm user))
         (remove #(nil? (:permission %)))))))
 
-(defn ^IPersistentMap uuid-for-path
```

Comment:
This is the same as a plain `path-stat` except it returns the UUID under `:uuid` rather than `:id`. I replaced it with calls to `path-stat`.